benchmark/competition/add_remove/mizu_test.go

```diff
@@ -0,0 +1,47 @@
+package addremove
+
+import (
+	"github.com/sedyh/mizu/pkg/engine"
+	"testing"
+)
+
+func BenchmarkIterMizu(b *testing.B) {
+	b.StopTimer()
+
+	_ = engine.NewGame(func(w engine.World) {
+		position := &Position{}
+		velocity := &Velocity{}
+
+		for i := 0; i < nEntities; i++ {
+			w.AddEntity(&position)
```

Comment:
This should not be a pointer (`position` is already a pointer). The effect is that everything below does nothing. Try it by counting `w.Each` loops iterations.

Reply:
Ok, I'll fix it.